[
exception_details
.
py
]
expected
:
if
(
os
=
=
"
linux
"
)
and
debug
and
(
version
=
=
"
Ubuntu
18
.
04
"
)
and
fission
:
[
OK
CRASH
]
[
test_exception_details
[
[
1
2
\
]
[
Symbol
.
iterator
\
]
(
)
-
expected28
\
]
]
bug
:
https
:
/
/
bugzilla
.
mozilla
.
org
/
show_bug
.
cgi
?
id
=
1853953
[
test_exception_details
[
'
mystring
'
[
Symbol
.
iterator
\
]
(
)
-
expected29
\
]
]
bug
:
https
:
/
/
bugzilla
.
mozilla
.
org
/
show_bug
.
cgi
?
id
=
1853953
[
test_exception_details
[
(
new
Set
(
[
1
2
\
]
)
)
[
Symbol
.
iterator
\
]
(
)
-
expected30
\
]
]
bug
:
https
:
/
/
bugzilla
.
mozilla
.
org
/
show_bug
.
cgi
?
id
=
1853953
[
test_exception_details
[
(
new
Map
(
[
[
1
2
\
]
\
]
)
)
[
Symbol
.
iterator
\
]
(
)
-
expected31
\
]
]
bug
:
https
:
/
/
bugzilla
.
mozilla
.
org
/
show_bug
.
cgi
?
id
=
1853953
[
test_exception_details
[
undefined
-
expected0
\
]
]
expected
:
if
(
os
=
=
"
linux
"
)
and
debug
and
(
version
=
=
"
Ubuntu
18
.
04
"
)
and
fission
:
[
PASS
ERROR
]
